ui/src/app/pages/signed-in/inactivity-monitor.tsx

```diff
@@ -184,21 +174,9 @@ export const InactivityMonitor = () => {
     };
   }, [authLoaded, isSignedIn]);
 
-  const onCloseInactivityModal = () => {
-    resetSignOutForInactivityTime();
-    // todo: clicking to close the modal will fire a click event, which will reset the timer,
-    // however, we should still reset the timers here so we don't rely on that implicit behavior
-  };
-
   if (!authLoaded || !isSignedIn) {
     return null;
   }
 
-  return (
-    <InactivityModal
-      closeFunction={onCloseInactivityModal}
-      currentTimeMs={currentTimeMs}
-      signOutForInactivityTimeMs={signOutForInactivityTimeMs}
-    />
-  );
+  return null;
```

Comment:
if this component always returns null, should it really be a component?

Reply:
I don't know. This is what copilit told me. Any better solutions?

Reply:
because it doesn't do anything visually anymore, maybe this makes more sense as a function now